arbnode/batch_poster.go

```diff
@@ -1257,6 +1251,8 @@ func (b *BatchPoster) maybePostSequencerBatch(ctx context.Context) (bool, error)
 		}
 	}
 
+	var firstNonDelayedMsg *arbostypes.MessageWithMetadata
+	var firstUsefulMsg *arbostypes.MessageWithMetadata
```

Comment:
Earlier, `firstMsgTime` was the header timestamp of `batchposition.MessageCount` and this remained the same even when we aborted batch building midway due to any reasons, but now the `firstUsefulMsg` and its corresponding `firstUsefulMsgTime` would change every time we abort building midway and resume it later on, as `b.building.msgCount` would have progressed. Is this intended?

Reply:
Great catch, I think this was causing the CI failure as well

Reply:
Fixed